x-pack/plugins/apm/public/components/app/RumDashboard/translations.ts

```diff
@@ -17,7 +17,7 @@ export const I18LABELS = {
     defaultMessage: 'Frontend',
   }),
   pageViews: i18n.translate('xpack.apm.rum.dashboard.pageViews', {
-    defaultMessage: 'Page views',
+    defaultMessage: 'Total Page views',
```

Comment:
```suggestion
    defaultMessage: 'Page views (total)',
```

We write the metric in parenthesis in many other places, so we might as well make it a consistent pattern.

Reply:
On the other hand, if we feel particularly strong about "Total page views" at least normalize the heading so Page is not capitalized.

Reply:
ahh that capitalise thing always haunts me